vnext/IntegrationTests/React.Windows.IntegrationTests.vcxproj

```diff
@@ -64,9 +64,6 @@
     <ProjectReference Include="..\Common\Common.vcxproj">
       <Project>{fca38f3c-7c73-4c47-be4e-32f77fa8538d}</Project>
     </ProjectReference>
-    <ProjectReference Include="..\ReactWindowsCore\ReactWindowsCore.vcxproj">
```

Comment:
why is this being deleted?

Reply:
Removed some unneeded dependencies that made the structure difficult to understand.

---
In reply to: [422402021](https://github.com/microsoft/react-native-windows/pull/4840#discussion_r422402021) [](ancestors = 422402021)